administration/src/mui-modules/application/forms/OrganizationForm.tsx

```diff
@@ -24,22 +25,22 @@ const WarningContactPersonSamePerson = styled(Alert)`
 
 const organizationCategoryOptions = {
   items: [
-    'Soziales/Jugend/Senioren',
-    'Tierschutz',
-    'Sport',
-    'Bildung',
-    'Umwelt-/Naturschutz',
-    'Kultur',
-    'Gesundheit',
-    'Katastrophenschutz/Feuerwehr/Rettungsdienst',
-    'Kirchen',
-    'Andere',
+    i18next.t('applicationForms:organizationCategories:social'),
```

Comment:
hm i realized that it may make sense to push the keys to the database and resolve them if we want to make them (for future issue) resolvable in different languages. Keys should then be moved to `application`